CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### New features and optimizations
 - Added public-facing official Dockerfile for NeMo-Aligner
 - Memory optimization in PPO that helps avoid OOM in the actor when sending training data to the critic
+- Added support for validation metrics that require token-by-token inference
```

Comment:
May just be me but "token-by-token inference" is a bit obscure to me. We should also mention this is just for SFT for now. Also we don't include any such metric right now. Suggestion (but feel free to improve):
```suggestion
- SFT: added support for custom validation metrics based on model generations
```

Reply:
I was using token-by-token to refer to the fact that we actually need to run inference generation, as opposed to just compute the loss which is done in one call to the model, while inference is a loop with as many calls as there are tokens. Not sure how to describe it more concisely, so I'll just go with your wording

Reply:
I think the word "generation" is more explicit. Again may just be me so feel free to let me know if you disagree!